composable_kernel/include/tensor_operation/xdlops_gemm.hpp

```diff
@@ -614,6 +614,43 @@ struct XdlopsGemm
                        Sequence<7>{}));
     }
 
+    template <typename CDesc_B_M0_N0_M1_N1_M2_N2>
+    __host__ __device__ static constexpr auto MakeCDescriptor_B_M0_N0_M1_N1_M2_M3_M4_N2(
```

Comment:
You don't need this function, because you don't need dimension "B" in wave wise GEMM

 B is already always 1, in block GEMM level